CHANGELOG.rst

```diff
@@ -77,6 +78,9 @@ Changed
 - Updated Mattermost connector documentation to be more clear.
 - Updated format strings to f-strings where appropriate.
 - Updated tensorflow requirement to ``1.15.0``
+
+Removed
```

Comment:
@btotharye Please don't resolve these comments unless you fix them! Can you please remove these added lines so that we can merge this then?

Reply:
I thought I had fixed them locally, I'll double check it

Reply:
ok this should now be resolved sorry, there was some confusion on my end with the changelog because I didn't add this changes I pulled in did somewhere along the line.

Thanks